src/components/Tool/FixMailingAddresses/Contact.tsx

```diff
@@ -0,0 +1,303 @@
+import React, { Fragment, ReactElement } from 'react';
+import {
+  Box,
+  Grid,
+  Button,
+  Typography,
+  Avatar,
+  makeStyles,
+  Hidden,
+} from '@material-ui/core';
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+import { Icon } from '@mdi/react';
+import { mdiCheckboxMarkedCircle, mdiLock, mdiPlus, mdiPencil } from '@mdi/js';
+import StarIcon from '@material-ui/icons/Star';
+import StarOutlineIcon from '@material-ui/icons/StarOutline';
+import theme from '../../../theme';
+import { emptyAddress } from './FixMailingAddresses';
+
+const useStyles = makeStyles(() => ({
+  left: {
+    [theme.breakpoints.up('md')]: {
+      border: `1px solid ${theme.palette.cruGrayMedium.main}`,
+    },
+  },
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: theme.spacing(2),
+    [theme.breakpoints.down('sm')]: {
+      border: `1px solid ${theme.palette.cruGrayMedium.main}`,
+    },
+  },
+  boxBottom: {
+    backgroundColor: theme.palette.cruGrayLight.main,
+    width: '100%',
+    [theme.breakpoints.down('xs')]: {
+      paddingTop: theme.spacing(2),
+    },
+  },
+  buttonTop: {
+    margin: theme.spacing(1),
+    [theme.breakpoints.down('sm')]: {
+      marginRight: theme.spacing(2),
+      marginTop: theme.spacing(2),
+      marginBottom: theme.spacing(2),
+    },
+    '& .MuiButton-root': {
+      backgroundColor: theme.palette.mpdxBlue.main,
+      color: 'white',
+    },
+  },
+  buttonIcon: {
+    marginRight: theme.spacing(1),
+  },
+  rowChangeResponsive: {
+    flexDirection: 'column',
+    [theme.breakpoints.down('xs')]: {
+      marginTop: -20,
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+  },
+  responsiveBorder: {
+    [theme.breakpoints.down('xs')]: {
+      paddingBottom: theme.spacing(2),
+      borderBottom: `1px solid ${theme.palette.cruGrayMedium.main}`,
+    },
+  },
+  paddingX: {
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+  },
+  paddingY: {
+    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+  },
+  paddingB2: {
+    paddingBottom: theme.spacing(2),
+  },
+  address: {
+    borderBottom: '1px solid gray',
+    width: '100%',
+    cursor: 'text',
+  },
+  hoverHighlight: {
+    '&:hover': {
+      color: theme.palette.mpdxBlue.main,
+    },
+  },
+  avatar: {
+    width: theme.spacing(7),
+    height: theme.spacing(7),
+  },
+}));
+
+export interface address {
+  source: string;
+  street: string;
+  locationType: string;
+  city: string;
+  state?: string;
+  zip: string;
+  country: string;
+  region?: string;
+  metro?: string;
+  primary: boolean;
+  valid: boolean;
+  new?: boolean;
```

Comment:
`new` is a reserved keyword in JS. At the very least it's throwing off the Github code formatting...